lesson20/index.js

```diff
@@ -0,0 +1,113 @@
+
+const body = document.querySelector("body");
+
+const createElementWithClassName = (type, className) => {
+  const element = document.createElement(type);
+  element.className = className;
+  return element;
+};
+
+const createLoading = () => {
+  const loading = createElementWithClassName("img", "loading");
```

Comment:
【thoughts】I was impressed! I learned that creating a "createElementWithClassName" makes it easy to create subsequent functions.